app/assets/javascripts/past_teams.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  var $selectedYear = $('select#past_teams');
```

Comment:
Really nice using data attributes for it!! 
I looked in your city.js but I couldn't find any view associated with it. I searched in the files and I put a breakpoint in your function to see the element. Is it used to auto-complete cities' names in forms? 
Anyway, I will replace the id by the data attribute.